src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingSliderLabel.tsx

```diff
@@ -57,13 +58,14 @@ const StakingSliderLabel = ({
       placement="top"
       popperOptions={tooltipOptions}
     >
-      {!enoughTokensToStakeMinimum ? (
+      {!enoughTokensToStakeMinimum && (
         <MinimumStakeMessage
           userMinStake={userMinStake}
           decimals={nativeTokenDecimals}
           symbol={nativeTokenSymbol}
         />
-      ) : (
+      )}
+      {remainingToStake !== '0' && (
```

Comment:
Could we have this condition be `enoughTokensToStakeMinimum && remainingToStake !== '0'`, to avoid the following:

![Screenshot (846)](https://github.com/JoinColony/colonyCDapp/assets/64402732/e7d2b9aa-08c1-423d-8326-00b3ba46c622)

I.e. they're showing up at the same time, which I originally wanted to avoid.